games/b.yaml

```
@@ -1085,6 +1085,24 @@
   images:
   - https://raw.githubusercontent.com/yannicka/bobby/master/src/assets/img/banner.png
 
+- name: Boing!
+  originals:
+  - Breakout
+  type: clone
+  repo: 'https://www.ibiblio.org/pub/micro/pc-stuff/Linux/games/arcade/#:~:text=boing'
+  url: 'https://www.ibiblio.org/pub/micro/pc-stuff/Linux/games/arcade/boing-0.1.lsm'
+  development: halted
+  status: playable
+  content: open
+  langs:
+  - C
+  frameworks:
+  - SVGALib
+  licenses:
+  - GPL1
```

```suggestion
  - GPL2
```
I can't find any mention of licenses in the source files, so I'm inclined to believe it's actually GPL2 as it is much more common than GPL1